docs/multibrokers.md

```diff
@@ -0,0 +1,218 @@
+
+   1. For Kafka the configurations you'll need to attain from your Kafka deployment are: 
+
+      - Address of Kafka's bootstrap servers
+      - The auto offset reset configuration
+
+   2. With these informations you can proceed by creating the configuration file to send Insprd. This file will require two more pieces of information:
+
+      - the image for that broker's sidecar, which Inspr provides to you 
+      - the address you want that sidecar to be deployed at.
+
+   The address by default should be `http://localhost` which just indicates that that deploy should happen on the same container as the other pieces of your dApp.
+      If you followed the Helm installation for Kafka suggested above, the file you need should be exactly like this:
+
+      `kafkaConfig.yaml`:
+
+      ```yaml
+      bootstrapServers: kafka.default.svc:9092
+      autoOffsetReset: earliest
+      sidecarImage: gcr.io/insprlabs/inspr/sidecar/kafka:latest
+      sidecarAddr: "http://localhost"
+      ```
+
+      If you choose to install the broker some other way, the information may change but the **format should remain like the example above**.
+
+      Finally if you are installing any broker other than Kafka you should consult the correct file format for that broker. 
+
+   3. Once you have you configuration file all you need to do is run:
+
+      ```shell
+      insprctl cluster config kafka relative_path_to/your_file.yaml
+      ```
+
+      To which the expected response is: `successfully installed broker on insprd`. 
+
+      This command works with the following pattern :
+
+      ```sh
+      insprctl cluster config <broker-name> <relative_file_path>
+      ```
+
+      Keep in mind:
+
+      - `<broker-name>` must be one of the supported broker options for Inspr. This is actually not an argument, it is a sub-command.
+      - `<relative_file_path>` must refer to a YAML file with the correct fields for the chosen broker.
+
+   #### This is all, you've successfully configured you Insprd to work with Kafka.
+
+
+
+## How does it work
+
+By configuring your broker what you are doing is subscribing the corresponding sidecar for that broker into a *Factory*, allowing Insprd to deploy sidecars of this type whenever a *dApp* you create uses a *Channel* configured to work with this broker. The architecture to support the dynamic choice of sidecars is based on a **Load-balancer** that is responsible for connecting to all of the broker specific sidecars your *dApp* demands.
+
+
+![Multibroker](./img/brokers.png)
+
+
+### Load Balancer
+
+The load balancer is technically itself a sidecar, since it is deployed alongside every *Node* and is used by it as an intermediate to all communication, but it serves different purposes. The load balancer is of central importance to the Multibroker architecture because of two of its functionalities: dynamic broker-specific sidecar balancing and serialization.
```

Comment:
`but it serves a different puporse` would be more fitting, no? 

Maybe just remove it and add the information in the previous sentence.